scripts/common

```diff
@@ -43,3 +43,44 @@ else
         '--remote_upload_local_results=false'
     )
 fi
+
+# kill_pid tries to kill the given pid(s), first softly then more aggressively.
+kill_pid() {
```

Comment:
Drive by comment: should we consider more seriously writing this logic in a small rust test that shells out to run other things (including Bazel and docker, if necessary)?

Reply:
Shell scripts are definitely awkward (I have to look up syntax and gotchas every time I mess with them), but are convenient for getting multi-process things (like integration tests) going.    

As [previously discussed](https://github.com/project-oak/oak/pull/116#issuecomment-503637388), in the past I found Python not to be that much of an improvement, but maybe Rust would do a better job.

(Even longer ago, I vaguely recall that Perl was decent for this sort of thing, but I doubt that would win any fans&hellip;)